<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IrcClient</class>
 <widget class="QMainWindow" name="IrcClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IrcClient</string>
  </property>
  <widget class="QWidget" name="chatWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLineEdit" name="sendEdit"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="sendBut">
      <property name="text">
       <string>send</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QListWidget" name="listWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="IrcClient.qrc"/>
 </resources>
 <connections/>
</ui>
